### PR TITLE
Create Dashboards 05, 06 and 07

### PR DESCRIPTION
**Pull request by lucasbois1**

### CODE REVIEW

**Comment by sadeghbarati** [comment]
Thanks for your time

Please note that you are not including `Dashboard05.vue` **new-york** style in this PR

Also, I would add other `Dashboard` examples in one PR

**Comment by lucasbois1** [comment]
> Thanks for your time
> 
> Please note that you are not including `Dashboard05.vue` **new-york** style in this PR
> 
> Also, I would add other `Dashboard` examples in one PR

Oh, I can add the **new-york** style too if you want

**Comment by sadeghbarati** [comment]
Please run `pnpm run build:registry` in `apps/www` directory again

File: apps/www/src/lib/registry/new-york/block/Dashboard05.vue [hunk 1]
```diff
@@ -0,0 +1,772 @@
...
+  MoreVertical,
+  Package,
+  Package2,
+  PanelLeft,
+  Search,
+  Settings,
+  ShoppingCart,
+  Truck,
+  Users2,
+} from 'lucide-vue-next'
```
Comment (sadeghbarati) [inline apps/www/src/lib/registry/new-york/block/Dashboard05.vue:25]: You should use `@radix-icons/vue` for new-york style

Comment (lucasbois1) [reply apps/www/src/lib/registry/new-york/block/Dashboard05.vue:25]: All **new-york** dashboards use icons from `'lucide-vue-next'`, should i change them too?

Comment (zernonia) [reply apps/www/src/lib/registry/new-york/block/Dashboard05.vue:25]: @sadeghbarati It's weird that the original repo is using `lucide` for `new-york` style too. I find it so inconsistent 😅

**Comment by zernonia** [comment]
> Thanks for your time
> 
> Please note that you are not including `Dashboard05.vue` **new-york** style in this PR
> 
> Also, I would add other `Dashboard` examples in one PR

Hi @LucasBois1  ! Let's add `Dashboard06` and `Dashboard07` in this PR too ya 😁

**Comment by lucasbois1** [comment]
> > Thanks for your time
> > Please note that you are not including `Dashboard05.vue` **new-york** style in this PR
> > Also, I would add other `Dashboard` examples in one PR
> 
> Hi @LucasBois1 ! Let's add `Dashboard06` and `Dashboard07` in this PR too ya 😁

Hey @zernonia 👋
Ok, I'll add those too here
Anything more to say about the icons?

File: apps/www/src/lib/registry/new-york/block/Dashboard05.vue (hunk 1 above)
Comment (zernonia) [reply apps/www/src/lib/registry/new-york/block/Dashboard05.vue:25]: Let's keep the icon same for now ya @LucasBois1  @sadeghbarati 😁
If `ui.shadcn` updates it in the future, we can do the same too

File: apps/www/src/lib/registry/new-york/block/Dashboard07.vue [hunk 2]
```diff
@@ -0,0 +1,629 @@
...
+                <CardHeader>
+                  <CardTitle>Product Details</CardTitle>
+                  <CardDescription>
+                    Lipsum dolor sit amet, consectetur adipiscing elit
+                  </CardDescription>
+                </CardHeader>
+                <CardContent>
+                  <div class="grid gap-6">
+                    <div class="grid gap-3">
+                      <Label html-for="name">Name</Label>
```
Comment (zernonia) [inline apps/www/src/lib/registry/new-york/block/Dashboard07.vue]: Let's update `html-for` to `for`, as Vue able to pass `for` attribute into native element, unlike React needed to use `html-for`.

Let's update all the occurance ya! 😁 Else LGTM!

**Review by zernonia — APPROVED** [review]
LGTM! Thanks for the PR @LucasBois1 ! 🤩

**Pull request merged**